src/sets.jl

```diff
@@ -30,6 +30,28 @@ julia> dimension(PositiveSemidefiniteConeTriangle(2))
 """
 function dimension end
 
+"""
+    dual_set(s::AbstractSet)
+
+Return the dual set of `s`, if there is no dual set returns an error.
```

Comment:
Please provide or reference a definition of a dual set.

Reply:
In MOI duality is defined in terms of conic duality so the dual set is the dual cone. Is this a good reference https://en.wikipedia.org/wiki/Dual_cone_and_polar_cone to put in documentation?